script.js

```diff
@@ -36,11 +36,12 @@ function generateQuote() {
     });
   tweetBtn.addEventListener("click", shareTweet);
 
-  shareTweet();
```

Comment:
- Removes `shareTweet()` call without click, as it triggers a popup which is blocked by browser popup-blockers anyway (eg. likely wont do anything)